client/boot/common.js

```diff
@@ -126,26 +126,6 @@ const loggedOutMiddleware = currentUser => {
 	}
 };
 
-const loggedInMiddleware = currentUser => {
```

Comment:
This `loggedInMiddleware` handles the `/` route for logged in users. Similarly, `loggedOutMiddleware` handler `/` redirections for logged out users. If only for special cases like desktop app or dev mode.

The `loggedOutMiddleware` code should be moved to the `root` section, too. The section then needs to have the `enableLoggedOut: true` flag and handle both logged-out and logged-in cases (the `isUserLoggedIn()` Redux selector will tell you the truth).